.github/workflows/kind.yml

```diff
@@ -108,7 +108,7 @@ jobs:
     - name: E2E Tests for Compliance Events API
       run: |
         make postgres
-        KUBECONFIG=${PWD}/kubeconfig_hub make e2e-test-coverage-compliance-events-api
+        make e2e-test-coverage-compliance-events-api
```

Comment:
I remove KUBECONFIG=${PWD}/kubeconfig_hub this. Because I use impersonate for auth. With KUBECONFIG=${PWD}/kubeconfig_hub, e2e-test is not working properly

Reply:
But..! is there any other way?

Reply:
It seems like github.com/stolostron/rbac-api-utils might also need `impersonate` permissions, so it might be hard to remove that...